Backend/Remora.Discord.API.Abstractions/API/Objects/Interactions/IInteraction.cs

```diff
@@ -122,4 +122,23 @@ public interface IInteraction
     /// Gets, for monetized apps, any entitlements for the invoking user.
     /// </summary>
     IReadOnlyList<IEntitlement> Entitlements { get; }
+
+    /// <summary>
+    /// Gets the context of the interaction.
+    /// </summary>
+    Optional<InteractionContextType> Context { get; }
+
+    /// <summary>
+    /// Gets the integrations that authorized the interaction.
+    /// </summary>
+    /// <remarks>
+    /// This is a mapping of the integration type to the ID of its resource.
+    /// <para>
+    /// The dictionary contains the following, given the circumstances: <br/>
+    /// - If the integration is installed to a user, a key of <see cref="ApplicationIntegrationType.UserInstallable"/> and the value is the user ID. <br/>
+    /// - If the integration is installed to a guild, a key of <see cref="ApplicationIntegrationType.GuildInstallable"/> and the value is the guild ID.
+    /// If the interaction is sent outside the context of a guild, the value is always zero.<br/>
```

Comment:
"value is always zero" - not sure what's meant by that. Please clarify.

Reply:
The value is...Zero? Here's [Discord's documentation](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-authorizing-integration-owners-object); outside of a guild, you're given a mapping that's effectively `"authorizing_integration_owners: { "0": 0 }`

Reply:
Ah, now I see. Thanks!